Doc/library/stdtypes.rst

```diff
@@ -802,6 +802,31 @@ number, :class:`float`, or :class:`complex`::
            hash_value = -2
        return hash_value
 
+.. _typebool:
+
+Boolean Type - :class:`bool`
+============================
+
+Booleans represent truth values. The :class:`bool` type has exactly two
+constant instances ``True`` and ``False``.
+
+Other types can be cast to bool explicitly using the function :func:`bool`.
```

Comment:
I think I preferred the old phrasing:

> The built-in function [bool()](https://docs.python.org/3/library/functions.html#bool) can be used to convert any value to a Boolean, if the value can be interpreted as a truth value (see section [Truth Value Testing](https://docs.python.org/3/library/stdtypes.html#truth) above)

It draws a clearer distinction between types and values and sidesteps the issue that `and` and `or` don't actually convert their operands to bool.

Reply:
don.